### PR TITLE
[Docs] Details on secrets when using private Git

### PR DESCRIPTION
**Pull request by jillnogold**

### CODE REVIEW

File: docs/projects/ci-integration.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ When the workflow runs inside the Git CI system it reports the pipeline progress
 
 <img src="../_static/images/git-pipeline.png" alt="mlrun-architecture" width="800"/><br>
 
+When working with a private Git repository, you need to create **GIT_TOKEN** secrets. For more details about creating secrets in mlrun see [MLRun-managed secrets](../secrets.html#mlrun-managed-secrets).
```
Comment (GiladShapira94) [inline docs/projects/ci-integration.md:30]: Need to fix the link seems to be broken

**Review by GiladShapira94 — COMMENTED** [review]
Looks good, Thanks

File: docs/projects/ci-integration.md (hunk 1 above)
Comment (jillnogold) [reply docs/projects/ci-integration.md:30]: link works in the build, but not in git

**Review by GiladShapira94 — COMMENTED** [review]
Approve

**Review by GiladShapira94 — APPROVED** [review]

**Pull request merged**